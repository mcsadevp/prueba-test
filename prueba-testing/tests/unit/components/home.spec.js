import { shallowMount } from '@vue/test-utils'
import HomeView from '@/views/HomeView.vue'
describe('Componente HomeView.vue', () => {
 test('Validación de match con el snapshot', () => {
 //Selección del sujeto de pruebas

 //Aserción
 })
})
describe('Componente HomeView.vue', () => {
    test('Validación de match con el snapshot', () => {
    //Selección el Sujeto de pruebas
    const wrapper = shallowMount(HomeView)
   
    //Aserción
    expect(wrapper.html()).toMatchSnapshot()
    })
   })
   