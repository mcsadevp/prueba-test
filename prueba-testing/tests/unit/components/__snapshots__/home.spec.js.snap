// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente HomeView.vue Validaci√≥n de match con el snapshot 1`] = `
<div class="home"><img alt="Vue logo" src="">
  <hello-world-stub msg="Welcome to Your Vue.js App"></hello-world-stub>
</div>
`;
